networking/common/obsolete.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# This is for the obsolete functions
+
+my_echo()
+{
+	echo "# $1" | tee -a "$OUTPUTFILE"
+	$1 | tee -a "$OUTPUTFILE"
+}
+
+# Usage : myecho "cmd" run check
+# 0 is ok, 1 is error
+myecho()
+{
+	echo -e "\n# $1" | tee -a "$OUTPUTFILE"
+	if [ $2 ];then
+		if [ "$2" == "run" ];then
+			eval $1
+			LASTSTATE=$?
+		else
+			echo -e "wrong option , should be 'run'\n"
+			return 1
+		fi
+	fi
+	if [ $3 ]; then
+		if [ "$3" == "check" ];then
+			if [ $LASTSTATE -ne 0 ];then
+				echo -e "Command # $1 failed\n" | tee -a "$OUTPUTFILE"
+				test_warn "Command_failed"
```

Comment:
Should this be a failure instead? Or can it mean the preparation for the test could have failed too (in which case it should abort)?